contracts/forwarding/astroport-forwarding/src/execute.rs

```diff
@@ -0,0 +1,122 @@
+use cosmwasm_std::{
+    coin, ensure, from_json, to_json_binary, DepsMut, Env, MessageInfo, Response, SubMsg, Uint128,
+    WasmMsg,
+};
+use cw20::{Cw20ExecuteMsg, Cw20ReceiveMsg};
+use euclid::{error::ContractError, token::TokenType};
+use forwarding::msgs::astroport::{Cw20HookMsg, SwapMsg};
+
+use astroport::router::ExecuteMsg as AstroportExecuteMsg;
+
+use crate::{
+    reply::ASTRO_SWAP_REPLY_ID,
+    state::{ForwardingState, FORWARDING_STATE, STATE},
+};
+
+pub fn execute_cw20_receive(
+    deps: &mut DepsMut,
+    env: &Env,
+    info: &MessageInfo,
+    msg: Cw20ReceiveMsg,
+) -> Result<Response, ContractError> {
+    // let state = STATE.load(deps.storage)?;
+    // let sender = msg.sender;
```

Comment:
The original sender won't be used for any potential refunds? If not we can delete those two comments